examples/guess-the-sketch/src/app.py

```diff
@@ -34,8 +42,36 @@
 
 headers = {"Content-Type": "application/json"}
 
-print("working directory: " + os.getcwd())
-print("files in cwd: " +  ', '.join(os.listdir(os.getcwd())))
+logger.info('gameserver started')
+logger.info('Agones SDK port: %s', PORT)
+
+body = Agones.SdkEmpty() # SdkEmpty
+agones = Agones.SDKApi(Agones.ApiClient(conf))
+agones.health(body)
+
+retry = 5
+while retry != 0:
+    try:
+        retry = retry - 1
+        agones.ready(body)
+        break
+    except:
+        time.sleep(2)
+        logger.info('retry connection')
+
+def agones_health():
+    while True:
+        try:
+            agones.health(body)
+            time.sleep(0.3)
+            logger.info('health check passed')
```

Comment:
This line is being printed in the logs very, very often. I think you could slow down the health interval a bit (maybe every second or two)?

Reply:
I have the impression that large interval for health check didn't work for Python client: https://github.com/googleforgames/agones/issues/2401. But I increased it to 2 and seems works, so updated it.